packages/oui-navbar/src/notification/navbar-notification.html

```diff
@@ -0,0 +1,119 @@
+<header class="oui-navbar-menu__header oui-navbar_mobile-only"
+    ng-if="::!!$ctrl.headerTitle">
+    <h3 class="oui-navbar-menu__title" ng-bind="::$ctrl.headerTitle"></h3>
```

Comment:
Shouldn't there be a `h1` and `h2` before ?

Reply:
It could even be a `<p>` or a `<span>`. 
It's not important here :)